README.md

```diff
@@ -5,48 +5,56 @@
-a company, which can yield dramatic effects in creativity, team building, longevity of
-employment, and in the creation of value—for the company, for its users and customers, and
-for the individuals who work to make it a success.
+work. In its ideal form, equity compensation aligns the interests of individual employees with the goals of
+the company they work for, which can yield dramatic results in team building, innovation,
+and longevity of employment.
+Each of these contributes to the creation of value—for the company, for its users and
+customers, and for the individuals who work to make it a success.
 
 Companies ranging from two-person startups to
 [the Fortune 500](http://fortune.com/fortune500/) have found that granting partial ownership
 in a company is among the best methods to attract and retain exceptional talent.
 In the United States,
 [since the 1950s](https://corpgov.law.harvard.edu/2014/10/02/what-has-happened-to-stock-options/),
-[stock options](#stock-options) have been a key part of pay for executives and other
-employees. In the United States, **7.2%** of all private sector employees (**8.5 million**
-people) and **13.1%** of *all* employees of companies with stock held stock options (from a
-2014 [NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf)). Many
-believe employee ownership has
+partial ownership through [stock options](#stock-options) has been a key part of pay for
+executives and other employees.
+In the United States, **7.2%** of all private sector employees (**8.5 million** people) and
+**13.1%** of *all* employees of companies with stock held stock options (from a 2014
+[NCEO analysis](https://www.nceo.org/assets/pdf/articles/GSS-2014-data.pdf)). Many believe
+employee ownership has
 [fostered innovations in technology](https://www.wsj.com/articles/reviving-the-flagging-spirit-of-silicon-valley-1428706671),
 especially in [Silicon Valley](https://en.wikipedia.org/wiki/Silicon_Valley), from the
 [early days of Hewlett-Packard](http://www.hp.com/hpinfo/abouthp/histnfacts/publications/measure/pdf/1976_07.pdf)
-to recent examples like [Facebook](https://en.wikipedia.org/wiki/Facebook). Dramatically,
-the first 3,000 employees of Facebook split roughly **$23 billion** at the time the company
-became public ([Financial Times](https://www.ft.com/content/d6599ae0-5738-11e1-869b-00144feabdc0)).
-
-Unfortunately, the ways equity is granted for compensation, including
-[restricted stock, stock options, and restricted stock units](#how-equity-is-granted), are
-notoriously complex, with confounding terminology and high-stakes decisions.
-If you talk to enough employees and hiring managers, and you’ll hear stories of how they
-or their colleagues met with the painful consequences of not learning enough up front.
-How an individual negotiates their equity-based compensation or when they choose to make
-certain decisions, such as exercise stock options, can have **major positive or negative**
```

Comment:
Open to it being a different phrase, but the intro is already a bit too long, so be good to preserve one or two boldfaced phrases in this paragraph. It helps the reader skim.

Reply:
Fixed.